examples/addons/aad-pod-identity/kubernetes-aad-pod-identity.json

```diff
@@ -0,0 +1,43 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "aad-pod-identity",
+            "enabled": true
+          }
+        ]
+      }
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_DS2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool",
+        "count": 3,
+        "vmSize": "Standard_DS2_v2",
+        "availabilityProfile": "VirtualMachineScaleSets"
```

Comment:
Need to support VMSS first...hold this PR until that is added or change this to `AvailabilitySet`

Reply:
Using AvailablitySet in this PR as the example. Will add the ScaleSet after adding the support for that in aad-pod-identity.